guides/common/modules/proc_benchmarking-raw-db-performance.adoc

```diff
@@ -5,7 +5,7 @@ ifndef::orcharhino[]
 To get a list of the top table sizes in disk space for both Candlepin, Foreman, and Pulp check https://github.com/RedHatSatellite/satellite-support/blob/master/postgres-size-report[postgres-size-report] script in https://github.com/RedHatSatellite/satellite-support[satellite-support] git repository.
 endif::[]
 
-PGbench utility (note you may need to resize PostgreSQL data directory `{postgresql-lib-dir}` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
+PGbench utility (note you may need to resize PostgreSQL data directory `/var/lib/pgsql` directory to 100 GiB or what does benchmark take to run) might be used to measure PostgreSQL performance on your system.
```

Comment:
I have introduced attributes for various directories in Foreman server. If you want to change it's values, please have a look at https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L79